pkg/gcv/validator.go

```diff
@@ -130,22 +108,18 @@ func loadYAMLFiles(dir string) ([]*configs.ConstraintTemplate, []*configs.Constr
 // NewValidator returns a new Validator.
 // By default it will initialize the underlying query evaluation engine by loading supporting library, constraints, and constraint templates.
 // We may want to make this initialization behavior configurable in the future.
-func NewValidator(options ...Option) (*Validator, error) {
-	ret := &Validator{}
-	for _, option := range options {
-		if err := option(ret); err != nil {
-			return nil, err
-		}
-	}
-	if ret.policyPath == "" {
+func NewValidator(policyPath string, policyLibraryPath string) (*Validator, error) {
```

Comment:
Is there a reason we're converting the options into args?

Reply:
Those were annoying me because in my mind they weren't technically options because if not set, the validator would return an error.  Best to let the compiler enforce the user passing the values, then sanity check the semantics.

Reply:
Fine by me.